routes/_components/profile/AccountProfileMeta.html

```diff
@@ -0,0 +1,132 @@
+<div class="account-profile-meta">
+   {@html meta}
```

Comment:
I think to avoid security issues with potential XSS attacks (as well as other issues with parsing HTML strings), it'd be more ideal to use a Svelte `{#each}` loop here